app/components/c-linklist.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <ul :class="b('')">
+    <li v-for="(value, index) in items" :class="b('item')" :key="index">
+      <a :class="b('link')"
+         :href="value"
+         target="_blank"
+      >
+        {{ value }}
+      </a>
+    </li>
+  </ul>
+</template>
+
+<script>
+  export default {
+    name: 'c-linklist',
+    // components: {},
+    // mixins: [],
+
+    props: {
+      /**
+       * Defines the items, displayed in the list.
+       */
+      items: {
+        type: Object,
+        required: true
+      }
+    },
+    // data() {
+    //   return {};
+    // },
+
+    // computed: {},
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    // methods: {},
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+  .c-linklist {
+    list-style-type: none;
+    padding-left: $spacing--0;
+
+    &__link {
+      padding-left: $spacing--0;
```

Comment:
- [ ] I would add here a word-break property, because the links can be very long and then it doesn't display the full text on mobile devices